.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
```

Comment:
this file is not required and before pushing make sure intended files are being added in a commit.